packages/components/src/index.ts

```diff
@@ -72,6 +74,7 @@ export * from './components/typography/Paragraph/Paragraph';
 export * from './components/typography/Text/Text';
 export * from './components/typography/TruncateWithTooltip/TruncateWithTooltip';
 export * from './components/Warning/Warning';
+export { Table } from './components/Table/Table';
```

Comment:
I think we should be able to use `Table` also without `Container`. And I think `Container` could be just `Card` component. It's basically the same, right? 

```tsx
<Card paddingType="none">
  <Table ... />
</Card>
```

Reply:
And one more thing - I think `TableRow` should have `hasBorder: boolean` prop